config/vendor-data/subjectsAndPaths.js

```diff
@@ -69,12 +68,43 @@ export const subjects = [
         } UNION {
           ?subject <http://www.w3.org/ns/prov#generated> ?formdata .
           ?formdata <http://purl.org/dc/terms/hasPart> ?remotefile .
-          ?remotefile <http://mu.semte.ch/vocabularies/core/uuid> ?remotefileUUID .
-          BIND (CONCAT("${HOSTNAME}files/", STR(?remotefileUUID), "/download") AS ?remotefileDownloadLink)
           ?remotefile ?ro ?rp .
         }
       `,
     },
+    post: {
+      delete: `
+        ?formdata <http://mu.semte.ch/vocabularies/ext/sessionStartedAtTime> ?sessionStartedAtTime .
+        ?formdata <http://mu.semte.ch/vocabularies/ext/decisionType> ?decisionType .
+        ?formdata <http://mu.semte.ch/vocabularies/ext/taxType> ?taxType .
+        ?formdata <http://mu.semte.ch/vocabularies/ext/taxRateAmount> ?taxRateAmount .
+        ?formdata <http://mu.semte.ch/vocabularies/ext/regulationType> ?regulationType .
+        ?remotefile <http://www.semanticdesktop.org/ontologies/2007/01/19/nie#url> ?originalDownloadLink .
+      `,
+      insert: `
+        ?formdata <http://lblod.data.gift/vocabularies/besluit/submission/form-data/sessionStartedAtTime> ?sessionStartedAtTime .
+        ?formdata <http://lblod.data.gift/vocabularies/besluit/submission/form-data/submissionType> ?decisionType .
+        ?formdata <http://lblod.data.gift/vocabularies/besluit/submission/form-data/taxType> ?taxType .
+        ?formdata <http://lblod.data.gift/vocabularies/besluit/submission/form-data/taxRateAmount> ?taxRateAmount .
+        ?formdata <http://lblod.data.gift/vocabularies/besluit/submission/form-data/regulationType> ?regulationType .
+        ?remotefile <http://www.semanticdesktop.org/ontologies/2007/01/19/nie#url> ?newDownloadLink .
+        ?remotefile <https://www.w3.org/TR/prov-o/#hadPrimarySource> ?originalDownloadLink .
```

Comment:
I think the IRI should be different:
```suggestion
        ?remotefile <http://www.w3.org/ns/prov#hadPrimarySource> ?originalDownloadLink .
```